src/rulesets/Base/base.jl

```diff
@@ -4,7 +4,7 @@
 
 @scalar_rule(abs(x::Real), sign(x))
 @scalar_rule(abs2(x), 2x)
-@scalar_rule(exp(x), Ω)
+@scalar_rule(exp(x::Real), Ω)
```

Comment:
Is `exp` really the only complex function that causes problems with Zygote? Or are the others just not tested?

Reply:
Testing is ongoing

Reply:
I'm pretty sure the tests will fail for all our complex functions. Zygote does something quite different there.

Reply:
turns out it only need conjugating on input and output, and othereise all is well.